src/Core/src/Platform/iOS/MauiView.cs

```diff
@@ -60,7 +60,13 @@ protected void CacheMeasureConstraints(double widthConstraint, double heightCons
 			_lastMeasureHeight = heightConstraint;
 		}
 
-		public ICrossPlatformLayout? CrossPlatformLayout
+		public override void SafeAreaInsetsDidChange()
+		{
+			base.SafeAreaInsetsDidChange();
+			View?.Handler?.UpdateValue("SafeAreaInsets");
```

Comment:
This string will be lost and magic will break. Can we use a constant somewhere for now if we are not sure that this is going to stay here for netFuture. Should we adda new property to ISafeAreaView? Is this property still something we want? At least it should be a const somewhere instead of a string that could be lost to time.

Reply:
This is in core, that BP is in controls.

Reply:
Hmmm yea, adding it to the interface is probably "the most correct" but since we want to take a longer look at all the `safearea` APIs a const is probably the better idea for now.

Reply:
What good does creating a constant for the one-and-only usage of this string do? It's a lame-duck API that we're going obsolete. It still has to match the name of the property in Controls, so there's no automated refactoring value; if that property name changes, we would just have to manually update the constant.